qtp_job_output_folder/tests/test_summary.py

```diff
@@ -61,36 +69,58 @@ def test_summary(self):
         # asserting content of html
         res = self.qclient.get("/qiita_db/artifacts/%s/" % aid)
         # cleaning artifact files, to avoid errors
-        [self._clean_up_files.extend([ff['filepath']])
-         for f in res['files'].values() for ff in f]
-        html_fp = res['files']['html_summary'][0]['filepath']
+        [
+            self._clean_up_files.extend([ff["filepath"]])
+            for f in res["files"].values()
+            for ff in f
+        ]
+        html_fp = res["files"]["html_summary"][0]["filepath"]
         with open(html_fp) as html_f:
             html = html_f.read()
 
-        self.assertCountEqual(
-            sorted(html.replace('<br/>', '').split('\n')),
```

Comment:
you dropped the sorting. Did you ensure in the generation of the summary, that ordering is stable?